### ISSUE TITLE
[Suggestion] Copper Pulsar - durability

### ISSUE THREAD

**Issue by bafb** [Safrodev/oxidized#41]
I would like it to have durability, because the point of this mod is to give more uses to copper, and in the game the copper pulsar has infinite durability, it feels somewhat buff and broken, it takes away the vanilla immersion.
In real life magnets demagnetize over time. I don't ask that they be demagnetized because it would be annoying, but I do ask that they simply break like a tool, and if the developer has the time and wants to improve it even more: that it can be enchanted like any vanilla tool.
It would be something more vanilla and balanced, I know that the copper pulsar doesn't change the gameplay that much, but if we think about it, it works like a rod to bringing items closer to you (and it also breaks). Personally, I would like to craft it from time to time or repair it with repair or unbreakability enchantments, since I always use it, and the experience would be more immersive.
In addition, copper and redstone really abound and it is what we players have the most surplus of and we have it stored in the chests without using it, and well, crafting is very cheap.

**Comment by Safrodev** [comment]
You're right, it is overpowered without durability. I will look into adding this as well.
